clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -395,46 +394,6 @@ func (pc *Controller) exposeAPIServerMetrics() error {
 	return nil
 }
 
-// runNodeExporter adds node-exporter as master's static manifest pod.
-// TODO(mborsz): Consider migrating to something less ugly, e.g. daemonset-based approach,
-// when master nodes have configured networking.
-func (pc *Controller) runNodeExporter() error {
-	klog.V(2).Infof("Starting node-exporter on master nodes.")
-	kubemarkFramework, err := framework.NewFramework(&pc.clusterLoaderConfig.ClusterConfig, numK8sClients)
-	if err != nil {
-		return err
-	}
-
-	// Validate masters first
-	nodes, err := client.ListNodes(kubemarkFramework.GetClientSets().GetClient())
-	if err != nil {
-		return err
-	}
-
-	var g errgroup.Group
-	numMasters := 0
-	for _, node := range nodes {
-		node := node
-		if util.LegacyIsMasterNode(&node) || util.IsControlPlaneNode(&node) {
```

Comment:
Do cloud environments even give access to the control plane nodes?

Reply:
I can understand the motivation for this to be a static pod however should we maintain this requirement? It's not hard to keep the daemonset and the static pods but they should be enabled with separate flags.